examples/oauth-v2/app.js

```diff
@@ -55,7 +55,7 @@ app.get('/slack/install', async (req, res, next) => {
 
 // example 1
 // empty callbackOptions Object
-// use slackOauth default success and failure methods
+// use default success and failure methods
```

Comment:
could we make a similar comment on L64 saying "using custom success and failure handlers", or something of that effect?